src/main/jbake/content/blog/2016/10/14/hawkular-apm-polyglot-zipkin.adoc

```diff
@@ -0,0 +1,200 @@
+= Hawkular APM Distributed Tracing of Polyglot Application using Zipkin Instrumentations
+Pavol Loffay
+2016-10-14
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, APM, Zipkin, polyglot, microservices, distributed tracing
+
+The latest focus of Hawkular APM has been in building a compatibility layer 
+for Zipkin instrumented applications. This enables Hawkular APM to leverage 
+Zipkin http://zipkin.io/pages/existing_instrumentations.html[instrumentation libraries]
+and instrument polyglot applications. 
+
+In this blog post we will look how to use Zipkin instrumentation libraries to instrument 
+polyglot environment and how the reported data looks in Hawkular APM and Zipkin server. 
+
+== Polyglot Application
+A typical polyglot application consists of multiple applications written in different programming
```

Comment:
Should the 2nd 'application', that denotes individual components be replaced with 'services' here and also below?